webpack/webpack-dev-server.js

```diff
@@ -21,15 +21,15 @@ module.exports = {
         SENTRY_PUBLIC_DSN: null,
         STEEMCONNECT_CLIENT_ID: JSON.stringify(process.env.STEEMCONNECT_CLIENT_ID || 'busy.app'),
         STEEMCONNECT_REDIRECT_URL: JSON.stringify(
-          process.env.STEEMCONNECT_REDIRECT_URL || 'http://localhost:3000/callback',
+          process.env.STEEMCONNECT_REDIRECT_URL || 'http://localhost:3000/callback'
```

Comment:
I see that you removed trailing commas (to fix issue with node older than 8). I don't have anything against about doing this locally, but I would rather not put it in PR (I highly recommend updating to node 8 though). Can you remove those changes from this PR?

Reply:
yes, you got it right. I will revert it back.